frontend/src/helpers/settings.ts

```diff
@@ -28,3 +28,8 @@ export const sentenceStateToColor: SentenceStateToColor = {
 export const OriginsGroupLabel = "Origins";
 export const ViasGroupLabel = "Vias";
 export const DestinationsGroupLabel = "Destinations";
+
+export const ChangeRequestStatus = {
```

Comment:
Note: This was a good idea, you should use this const instead of the hardcoded values